README.md

```diff
@@ -58,6 +58,37 @@ default unless you set the `QT_API` environment variable.
 * `pyside6` (to use PySide6).
 
 
+### Module aliases and constants
+
+* `QtCore.pyqtSignal`, `QtCore.pyqtSlot` and `QtCore.pyqtProperty` (available on PyQt5/6) are instead exposed as `QtCore.Signal`, `QtCore.Slot` and `QtCore.Property`, respectively, following the Qt5 module layout.
```

Comment:
```suggestion
* `QtCore.pyqtSignal`, `QtCore.pyqtSlot` and `QtCore.pyqtProperty` (available on PyQt5/6) are instead exposed as `QtCore.Signal`, `QtCore.Slot` and `QtCore.Property`, respectively, following the Qt5 and PySide2/6 module layout.
```

Maybe better to say this then?

Reply:
I think just leaving `Qt5 module layout` is enough